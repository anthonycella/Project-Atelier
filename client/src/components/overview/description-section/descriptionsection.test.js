import renderer from 'react-test-renderer';
import DescriptionSection from './descriptionsection.jsx';

it('Description Section renders to the page', () => {
  const component = renderer.create(
    <DescriptionSection />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
