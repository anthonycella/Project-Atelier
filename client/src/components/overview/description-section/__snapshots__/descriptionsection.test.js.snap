// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Section renders to the page 1`] = `
<div
  className="sc-jSMfEi ceClUd"
>
  <div
    className="sc-dkzDqf iECctP"
  >
    <div
      className="sc-bczRLJ cgftLq"
    >
      Catchphrase
    </div>
    <div
      className="sc-gsnTZi fRjVx"
    >
      According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees dont care what humans think is impossible.
    </div>
  </div>
  <div
    className="sc-eCYdqJ gJgfQB"
  >
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
    <div
      className="sc-hKMtZM efARqR"
    >
      I am one with the force and the force is with me
    </div>
  </div>
</div>
`;
